tinygrad/runtime/ops_metal.py

```diff
@@ -60,13 +64,17 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
 class MetalAllocator(LRUAllocator):
   def __init__(self, device:MetalDevice):
     self.device:MetalDevice = device
+    self.i = 0
     super().__init__()
   def _alloc(self, size:int) -> Any:
     ret = self.device.device.newBufferWithLength_options_(size, Metal.MTLResourceStorageModeShared)
     if ret is None: raise MemoryError(f"Metal OOM while allocating {size=}")
+    ret.setLabel_(f"{self.device.dname} buf {(self.i)}")
```

Comment:
Is this slow at all?

Reply:
0.021209 milliseconds (21042 nanoseconds measured with time.perf_counter_ns())